config/style/topographic.json

```diff
@@ -7,7 +7,7 @@
     "LINZ Basemaps": { "type": "vector", "url": "/v1/tiles/topographic/EPSG:3857/tile.json", "attribution": "© 2022 Toitū Te Whenua - CC BY 4.0" },
     "LINZ-Texture-Relief": { "type": "raster", "tiles": ["/v1/tiles/texturereliefshade/EPSG:3857/{z}/{x}/{y}.webp"], "tileSize": 256, "minzoom": 0, "maxzoom": 20 }
   },
-  "sprite": "/sprites/topographic",
+  "sprite": "/v1/sprites/topographic",
   "glyphs": "https://orangemug.github.io/font-glyphs/glyphs/{fontstack}/{range}.pbf",
```

Comment:
this needs to be `/v1/fonts/`

Reply:
Do we have all the fonts available for the topographic map now? If so, we should switch this.